config/env/test.example.env

```diff
@@ -0,0 +1,30 @@
+# Application
+CONFIG_SECRET_KEY_BASE: ""
+
+# Cloudinary
+CLOUDINARY_CLOUD_NAME=""
+CLOUDINARY_API_KEY=""
+CLOUDINARY_API_SECRET=""
+
+# Configuration values for sending email, set these to true, configure your default "from address",
+# and configure smtp.yml in order to send email
+SEND_EMAIL: "false"
```

Comment:
If you want to use a SEND_EMAIL key, you need to use like: `SEND_EMAIL=false` and declare ENV['SEND_EMAIL'] on destination! ;D